common/src/main/java/dev/latvian/mods/kubejs/core/ModifiableItemKJS.java

```diff
@@ -0,0 +1,28 @@
+package dev.latvian.mods.kubejs.core;
+
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.ImmutableMultimap;
+import com.google.common.collect.Multimap;
+import net.minecraft.world.entity.ai.attributes.Attribute;
+import net.minecraft.world.entity.ai.attributes.AttributeModifier;
+import org.apache.commons.lang3.NotImplementedException;
+
+public interface ModifiableItemKJS {
+	default Multimap<Attribute, AttributeModifier> getAttributeMapKJS() {
+		throw new NotImplementedException("A mixin should have implemented this method!");
+	}
+
+	default void setAttributeMapKJS(Multimap<Attribute, AttributeModifier> attributes) {
+		throw new NotImplementedException("A mixin should have implemented this method!");
```

Comment:
This erroring default implementation is only needed at compile time if the interface is injected via Loom, otherwise you can just leave this as an abstract method

Reply:
I copied `ItemKJS`'s code before. The `getAttributeMapKJS` and the `setAttributeMapKJS` are both abstract now.